sdk/blueprint/Microsoft.Azure.Management.Blueprint/src/Properties/AssemblyInfo.cs

```diff
@@ -11,8 +11,8 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
-[assembly: AssemblyVersion("0.20.0.0")]
-[assembly: AssemblyFileVersion("0.20.0.0")]
+[assembly: AssemblyVersion("0.20.5.0")]
```

Comment:
AssemblyVersion need not be updated unless it's a breaking change.

Reply:
Done